content/docs/packaging-an-application/preflight-checks.md

```diff
@@ -81,4 +81,10 @@ dismissing the warnings. Upon dismissing preflight warnings, an entry will be re
 {{% page_notes %}}
 - `min_disk_space` does not guarantee free space, it refers to the disk size mounted at the specified location.
 - The requested `docker_version` must be one of the versions Replicated supports.
-{{% /page_notes %}}
+
+- Currently supported versions are - {{% /page_notes %}}
+ 
+| **Scheduler** | **Min Docker Version** | **Max Docker Version** |
```

Comment:
i think this would be better as just a sentence in the note rather than a table

Reply:
haha i was debating that in my mind, you're right it should just be a simple sentence !